### PR TITLE
Testing Tic Tac Toe - Project Implementation

### PR DESCRIPTION
**Pull request by ermin-cahtarevic**
Dear TSE,

In this project, we 
- [x] added tests for all the public functions in our game logic
- [x] added tests in a separate folder
- [x] used ES6 
- [x] added Webpack to the project structure
- [x] refactored the code
- [x] added Babel so tests can use import and export keywords

We hereby welcome your review.

### CODE REVIEW

File: src/dom/game.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  const clearBoard = (board) => {
+    board = ['', '', '', '', '', '', '', '', ''];
+    return board;
+  };
+
+
+  return {
+    board, checkWins, checkDraws, clearBoard,
+  };
+})();
```
Comment (azdrenymeri) [inline src/dom/game.js]: Why this is an iffy function :thinking:

File: src/dom/main.js [hunk 2]
```diff
@@ -62,9 +100,10 @@ const displayController = (() => {
     toggleGame.classList.toggle('d-none');
     playerNames.innerHTML = `
       ${players[0]} vs ${players[1]}`;
+    play();
   };
 
   return { display };
 })();
```
Comment (azdrenymeri) [inline src/dom/main.js]: Same here.
Why did you convert this to an iffy function :thinking:

File: __tests__/game.test.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+test('it returns true for a full board without winning combinations', () => {
+  const newBoard = ['X', 'O', 'X', 'X', 'O', 'O', 'O', 'X', 'X'];
+  expect(gameBoard.checkDraws(newBoard)).toEqual(true);
+});
+
+// tests clearBoard function
+test('it clears the board', () => {
+  const newBoard = ['X', 'O', 'X', 'X', 'O', 'O', 'O', 'X', 'X'];
+  expect(gameBoard.clearBoard(newBoard)).toEqual(['', '', '', '', '', '', '', '', '']);
+});
```
Comment (azdrenymeri) [inline __tests__/game.test.js]: It is very common to use `describe` blocks to group related tests together
[ jestjs.io Docs(Describe Block) ](https://jestjs.io/docs/en/api#describename-fn)
[This](https://dev.to/valentinogagliardi/jest-tutorial-for-beginners-getting-started-with-jest-for-javascript-testing-e6c) is a great article to learn more about testing.

**Review by azdrenymeri — APPROVED** [review]
### Hello @ermin-cahtarevic and @christianotieno :wave:
### The design looks nice :+1: 
### There are no further requirements and you are free to merge it :heavy_check_mark:

**Comment by ermin-cahtarevic** [comment]
@azdrenymeri Thank you for the review, and for your kind suggestions. We have implemented the changes.

**Pull request merged**